src/parser/semantic_check.c

```diff
@@ -13761,7 +13761,7 @@ pt_check_path_eq (PARSER_CONTEXT * parser, const PT_NODE * p, const PT_NODE * q)
 	{
 	  return 1;
 	}
-      break;
```

Comment:
Not related to the issue. How about putting the following condition first?
```
if (p->info.name.spec_id != q->info.name.spec_id)
	{
	  return 1;
	}
```

Reply:
I agree. I will change it.